src/react/projects/spark-react/src/matchMedia.js

```diff
@@ -0,0 +1,11 @@
+// window.matchMedia is referenced by a utility function in spark-core. This function
```

Comment:
I think you mentioned that you were removing this file yesterday. Did you mean to leave this in here?

Reply:
yeah I haven't moved it yet. matchMedia is only for the react tests and _setup.js is in the folder with the vanilla tests so they may not be able to be combined.